### PR TITLE
[SMALLFIX] Fix FileSystemMasterIntegrationTest [WIP]

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -373,7 +373,7 @@
             <argLine>-Djava.net.preferIPv4Stack=true -XX:MaxPermSize=512M</argLine>
             <redirectTestOutputToFile>${test.output.redirect}</redirectTestOutputToFile>
             <!-- Alphabetical order on even hours, reverse-alphabetical on odd hours  -->
-            <runOrder>hourly</runOrder>
+            <runOrder>reversealphabetical</runOrder>
```
Comment (gpang) [inline pom.xml]: Does this have to be modified from `hourly`?

File: tests/src/test/java/alluxio/worker/DataServerIntegrationTest.java [hunk 2]
```diff
@@ -106,6 +106,7 @@ public final void before() throws Exception {
   @After
   public final void after() throws Exception {
     mBlockMasterClient.close();
+    BlockStoreContext.get().releaseWorkerClient(mBlockWorkerClient);
```
Comment (ifcharming) [inline tests/src/test/java/alluxio/worker/DataServerIntegrationTest.java]: Shall we also add similar cleanup in `BlockServiceHandlerIntegrationTest` ?

Comment (peisun1115) [inline tests/src/test/java/alluxio/worker/DataServerIntegrationTest.java]: I don't see we acquire block worker client in that test. I was looking at BlockServiceHandlerIntegrationTest.java

**Comment by peisun1115** [comment]
Ok. It doesn't work on jenkins then. I will mark it as WIP and keep working on it.

**Comment by peisun1115** [comment]
@gpang @ifcharming I am closing this. Please review https://github.com/Alluxio/alluxio/pull/4024 instead. Thank you.

**Pull request closed without merging**